### ISSUE TITLE
great but change folder/file option

### ISSUE THREAD

**Issue by mtissington** [zzhanghub/alfred-template-file#1]
This is great but sometimes I need to create a file without an extension.
I see it's possible to modify the workflow ... maybe check instead to see if it begins with ./ ?

**Comment by Pa7rickStar** [comment]
@mtissington I agree! I created a version which will ask you if you want to create a directory or a file without an extension. It also allows you to have multiple templates per file type.

See my [PR here](https://github.com/zzhanghub/alfred-template-file/pull/3#issue-2918037116).
